swift-sdk/Internal/IterableAPIInternal.swift

```diff
@@ -315,6 +326,15 @@ final class IterableAPIInternal: NSObject, PushTrackerProtocol, AuthProvider {
                                  forResult: result)
     }
     
+    func trackInAppClose(_ message: IterableInAppMessage, location: InAppLocation = .inApp, inboxSessionId: String? = nil, source: InAppCloseSource? = nil, clickedUrl: String? = nil) {
+        let result = apiClient.track(inAppClose: InAppMessageContext.from(message: message, location: location, inboxSessionId: inboxSessionId),
+                                     source: source,
+                                     clickedUrl: clickedUrl)
+        IterableAPIInternal.call(successHandler: IterableAPIInternal.defaultOnSucess(identifier: "trackInAppClose"),
+                                 andFailureHandler: IterableAPIInternal.defaultOnFailure(identifier: "trackInAppClose"),
+                                 forResult: result)
+    }
+    
```

Comment:
Just add extra parameter `inboxSessionId` to existing method with default value of nil.